setup/connector_algolia/setup.py

```diff
@@ -5,7 +5,7 @@
     odoo_addon={
         'external_dependencies_override': {
             'python': {
-                'algoliasearch': 'algoliasearch>=1.0.0,<2.0.0',
+                'algoliasearch': 'algoliasearch>=2.0,<3.0',
```

Comment:
In v13+ the version specifier can be declared in the manifest external dependencies.

Reply:
done